frontend/package.json

```diff
@@ -92,7 +92,6 @@
     "test:server:coverage": "cd ./server && npm test -- --coverage && cd ..",
     "test:coverage": "npm test -- --coverage && npm run test:server:coverage",
     "test:ci": "export CI=true && npm run format:check && npm run lint && npm run test:coverage",
-    "test:ci:prow": "npm set unsafe-perm true && npm ci && npm run test:ci && ./scripts/report-coveralls.sh",
```

Comment:
We can also remove the `./scripts/report-coveralls.sh` and `frontend/scripts/get-coveralls-repo-token.js` files. Seems like they are no longer referenced with this change.

Reply:
Done, removed these files.